storage/Devices/EncryptionImpl.cc

```diff
@@ -294,6 +312,35 @@ namespace storage
 
 
     Text
+    Encryption::Impl::do_rename_etc_crypttab_text(const Device* lhs, Tense tense) const
```

Comment:
That function name implies that the /etc/crypttab file is renamed, but that's not true: Entries **in** that file are renamed, so it should rather be `do_rename_in_etc_crypttab`. Similar for all the functions of this type.